core/tests/performance_framework/perf_domain.py

```diff
@@ -144,3 +144,33 @@ def get_dom_ready_time_millisecs(self):
     def get_request_time_millisecs(self):
         """Returns the total request time (in milliseconds)."""
         return self._get_duration_millisecs('responseEnd', 'requestStart')
+
+
+class MultiplePageSessionMetrics(object):
+    """Domain object for multiple PageSessionMetrics.
+    """
+
+    def  __init__(self, page_session_metrics):
+        self.page_metrics = page_session_metrics
+        self._validate()
+
+    def _validate(self):
+        if not isinstance(self.page_metrics, list):
+            raise utils.ValidationError(
+                'Expected page_session_metrics to be a list, '
+                'received %s' % self.page_metrics)
+
+    def get_page_load_time_millisecs(self):
```

Comment:
Perhaps ``get_average_page_load_time_millisecs``? Ditto elsewhere, since that's what these are returning.